yarn.lock

```diff
@@ -8868,13 +8858,6 @@ gauge@~2.7.3:
     strip-ansi "^3.0.1"
     wide-align "^1.1.0"
 
-gaze@^1.0.0:
```

Comment:
was this automatic? seems to be added for grunt.

Reply:
It was added automatically.